python/tk_multi_publish2/ui/dialog.py

```diff
@@ -58,12 +58,14 @@ def setupUi(self, Dialog):
 "}")
         self.large_drop_area_label.setAlignment(QtCore.Qt.AlignCenter)
         self.large_drop_area_label.setObjectName("large_drop_area_label")
+        self.large_drop_area_label.setVisible(Dialog.manual_load_enabled)
```

Comment:
We can't modify this file by hand. It is automatically generated when the `.ui` designer file is built.